deploy/tasks/00-cosmosdb.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * Run this task to deploy CosmoDB database and collections:
+ * ts-node cosmosdb.ts
+ * 
+ * This task assumes that the following resources are already created:
+ *  - Resource group
+ *  - CosmoDB database account
+ */
+// tslint:disable:no-console
+// tslint:disable:no-any
+
+import * as config from "./../tfvars.json";
+import { login } from "./login";
+
+import CosmosDBManagementClient = require("azure-arm-cosmosdb");
+import * as documentdb from "documentdb";
+
+const DocumentClient = documentdb.DocumentClient;
+
+const collectionNotExists = (
```

Comment:
usually better to have checks in the positive (`collectionExists`)

Reply:
yes that was my first try, but then the calling code was: `collectionExists().then()` 
while it is really `collectionNotExists().then()`